__tests__/integration.test.js

```diff
@@ -38,6 +38,33 @@ describe('/api/topics', () => {
   });
 });
 
+describe('/api/articles', () => {
+  test('GET 200: sends an array of articles to the client ', () => {
+    return request(app)
+      .get('/api/articles')
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        expect(articles).toHaveLength(13);
```

Comment:
Nice - this length check does a couple of things 1 protects a false positive from a foreach on an empty array and 2 obviously assert you have received the expected number of articles 👍